account_payment_notification/views/account_payment_views.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 Moduon Team S.L.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+<data>
+
+    <record id="action_notifiy_mail_single" model="ir.actions.act_window">
+        <field name="name">Notifiy by email</field>
+        <field
+            name="context"
+            eval="{
+                'account_payment_notification': 'single',
+                'custom_layout': 'mail.message_notification_email',
+                'default_template_id': ref('mail_template_notification'),
+                'default_use_template': True,
+                'mail_post_autofollow': True,
+                'mailing_document_based': True,
+            }"
+        />
+        <field name="binding_model_id" ref="account.model_account_payment" />
+        <field name="binding_view_types">form</field>
+        <field name="res_model">mail.compose.message</field>
+        <field name="target">new</field>
+        <field name="view_mode">form</field>
+    </record>
+
+    <record id="action_notifiy_mail_multi" model="ir.actions.act_window">
+        <field name="name">Notifiy by email</field>
```

Comment:
Can't do that, because the wizard is different. However, the user can use the provided filters to choose for example payments for customers that have no email but have phone and notify only those by SMS.